src/main/java/com/wolox/training/services/BookService.java

```diff
@@ -0,0 +1,76 @@
+package com.wolox.training.services;
+import com.wolox.training.exceptions.BadRequestException;
+import com.wolox.training.exceptions.NotFoundException;
+import com.wolox.training.exceptions.ServerErrorException;
+import com.wolox.training.repositories.BookRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import com.wolox.training.models.Book;
+
+import java.util.List;
+
+@Service
+public class BookService {
+    @Autowired
+    BookRepository bookRepository;
+
+    public Book createBook(Book book){
+        if(!checkBookFields(book)){
+            throw new BadRequestException("Invalid book fields");
+        }
+        try {
+            return bookRepository.save(book);
+        }catch(ServerErrorException serverError){
+            throw new ServerErrorException("Internal server error");
+        }
+    }
+
+    public List<Book> getBooks(){
+        try {
+            return bookRepository.findAll();
+        }catch(ServerErrorException serverError){
+            throw new ServerErrorException("Internal server error");
```

Comment:
We should use a class that contains the error messages as constants, we should not have them literal in the place where the exception is raised